<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="LineLoading" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>531</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="WBSwitchButton" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WBSwitchButton</class>
   <extends>QWidget</extends>
   <header location="global">wbswitchbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineLoading</class>
   <extends>QWidget</extends>
   <header location="global">lineloading.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
